modes/longitudinal/src/index.js

```diff
@@ -123,15 +123,15 @@ function modeFactory({ modeConfiguration }) {
       const modalities_list = modalities.split('\\');
 
       // Slide Microscopy modality not supported by basic mode yet
-      return !modalities_list.includes('SM');
+      return !!modalities_list.filter(modality => modality != 'SM' && modality !== 'ECG').length;
```

Comment:
Fixing the test condition to work correctly - it allows applying the mode as long as it contains at least one other modality.